build.sbt

```diff
@@ -17,7 +17,7 @@
 import java.nio.file.Files
 import TestParallelization._
 
-val sparkVersion = "3.3.1"
+val sparkVersion = "3.3.0"
```

Comment:
does this have any downstream consequences?

this requires that all our production code have "provided" spark dependencies, right?

is there any other way to test these changes with spark 3.3.0? e.g. an integration test?

Reply:
Sorry could've also clarified in the PR description https://github.com/delta-io/delta/pull/1633#issuecomment-1457311946

Not merging this change, it's just to run the tests in github actions. But maybe we should run our integration tests with various patch versions

Reply:
https://github.com/delta-io/delta/pull/1633/files/322df2c04584b9ce32c32d33a031eee4a7795c84..98c91ddd70c3709691a1fcf83b055266ae889762 changes to be merged